examples/hosts/app-integration/external-data/.eslintrc.js

```diff
@@ -9,19 +9,7 @@ module.exports = {
         "prettier",
     ],
     rules: {
-        // TODO: remove once dependency on base config has been updated.
-        "@typescript-eslint/explicit-member-accessibility": [
-            "error",
-            {
-                accessibility: "explicit",
-                overrides: {
-                    accessors: "explicit",
-                    constructors: "explicit",
-                    methods: "explicit",
-                    properties: "explicit",
-                    parameterProperties: "explicit",
-                },
-            },
-        ],
+        // TODO: Enable and add missing docs
+        "jsdoc/require-jsdoc": "off",
```

Comment:
I plan on handling this separately since I am currently working in this space :)